src/app/utils/field-validation/validators/__tests__/email-validation.spec.ts

```diff
@@ -1052,131 +1052,4 @@ describe('Email field validation V3', () => {
       new ValidateFieldError('Invalid answer submitted'),
     )
   })
-
```

Comment:
These TC are removed as they are migrated to the test for checkIsResponseChangedV3 instead